public/js/index.js

```diff
@@ -594,6 +594,7 @@ function initWikiForm() {
                         function (data) {
                             preview.innerHTML = '<div class="markdown">' + data + '</div>';
                             emojify.run($('.editor-preview')[0]);
+                            $('.editor-preview').autolink();
```

Comment:
@lunny (2nd question) no, as you see below, it is for both editor preview and final rendering. Reasoning is shown above.